x-pack/plugins/ml/public/components/annotations_table/annotations_table.js

```diff
@@ -268,6 +284,14 @@ class AnnotationsTable extends Component {
       },
     ];
 
+    const jobIds = _.uniq(annotations.map(a => a.job_id));
+    if (jobIds.length > 1) {
+      columns.unshift({
+        field: 'job_id',
+        name: 'job ID'
```

Comment:
Any reason why you haven't made this column sortable?

Reply:
Thanks for spotting this, was an oversight. Fixed in 5bf8e69303.